CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 0.2.11
+ * Add the "force" parameter to the obtainAccessCredentialsViaUserConsent API.
```

Comment:
use `force` instead of "force", indicate that it's optional. Also, use `obtainAccessCredentialsViaUserConsent`

Reply:
Changed using `force`.